src/systemctl/systemctl-edit.c

```diff
@@ -108,78 +108,64 @@ static int get_file_to_edit(
                 const LookupPaths *paths,
                 const char *name,
                 char **ret_path) {
-
```

Comment:
Oh, please do not drop these empty lines.

Reply:
The comment is for the 4th commit. You unnecessarily removed empty lines in the 4th commit, and re-add some of them in a later commit...

Reply:
Oops, sorry